hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemRandomRead.java

```diff
@@ -602,7 +602,7 @@ public void testReadBlob() throws IOException {
       Mockito.verify(mockClient, Mockito.atLeast(1)).getBlobProperty(
               Mockito.any(Path.class), Mockito.any(TracingContext.class));
     } else {
-      Mockito.verify(mockClient, Mockito.times(1)).getPathStatus(
+      Mockito.verify(mockClient, Mockito.times(2)).getPathStatus(
```

Comment:
Is it expected that getPathStatus() should be called twice??

Reply:
Yes